sensor_msgs/PointCloud2 cloud_in
---
SegmentedPointCloud segmented_scene
